ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/vlan_interfaces.schema.yml

```diff
@@ -94,6 +94,30 @@ keys:
                 description: VRF where DHCP server can be reached
                 convert_types:
                   - int
+        ipv6_helpers:
```

Comment:
Need to move this key down so it is in order with the other ipv6 keys

Reply:
done